mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/dao/DaoImplementationGenerator.java

```diff
@@ -107,6 +116,39 @@ protected String getFileName() {
     classBuilder.addMethod(initBuilder.build());
     classBuilder.addMethod(constructorBuilder.build());
 
+    documentEntityConstants(classBuilder);
+
     return JavaFile.builder(implementationName.packageName(), classBuilder.build());
   }
+
+  /**
+   * If the DAO has a constant of a mapped entity, dump the definition of that entity in the DAO's
+   * javadoc.
+   *
+   * @deprecated this is a quick and dirty test to check that entity parsing works, currently the
+   *     only way because we don't use entities in generated code yet. TODO delete
+   */
+  @Deprecated
+  private void documentEntityConstants(TypeSpec.Builder classBuilder) {
```

Comment:
Example of test code:
```java
@Entity
public class Product {
  // fields, getters, setters
}
@Dao
public interface ProductDao {
  Product TEST = null;
}
```
Then look at `ProductDao_Impl.java`.

This is a bit lame but our DAOs do nothing else right now, so this can be used as a smoke test until we get a little further.

Reply:
> Its initAsync method doesn't seem to do much for now, is this just a method stub?

OK I think this is just a stub according to the design doc.

Reply:
> It seems that the import of CqlIdentifier is missing.

Have you fetched recently? I fixed that in 142621e3c yesterday.

> Its initAsync method doesn't seem to do much for now

Yes, it's just a stub. Eventually it will prepare all the underlying statements asynchronously, and chain the construction of the DAO to the completion of those prepare requests.

Reply:
My bad, I fixed the `CqlIdentifier` bug on `java2078`, it's not on this branch. I can rebase the PR if that's OK with you.